composer.json

```diff
@@ -20,7 +20,7 @@
 		"php": ">=7.1.0",
 		"symfony/process": "^4.2",
 		"symfony/dependency-injection": "^4.2",
-		"james-heinrich/getid3": "^1.9",
+		"james-heinrich/getid3": "dev-master",
```

Comment:
I am assuming because of the Hotfix/Patch you mentioned in chat ?

Reply:
Yes, exactly.